config/jest/vite.config.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
```

Comment:
I found this to be a bit confusing as it's not apparent at first that vite is run by jest - adding a comment here explaining what vite is and how it's being run would be helpful.